src/components/SelectionArea.js

```diff
@@ -24,8 +24,9 @@ function getTextStyle(data) {
     const size = {fontSize: data.fontSize}
     const color = {color: data.textColor}
     const fontFamily = {fontFamily: data.fontFamily}
+    const showWhitespaces = { whiteSpace: "pre" }
 
-    return Object.assign({}, size, color, fontFamily)
+    return Object.assign({}, size, color, fontFamily, showWhitespaces)
```

Comment:
Let's make that style object more generic. For example:

```
const baseStyles = { whiteSpace: "pre" }
return Object.assign(baseStyles, size, color, fontFamily)
```

Reply:
great, idea, adding a follow up commit right now